### DISCUSSION TITLE
Lista Ceu Azul

### DISCUSSION THREAD

**Discussion by lucasherrrera** [anacsv/tech-br-em-pt#19] [Geral]
Lucas Herrera - lsfherrera@hotmail.com
Tammy Vitorino - tamara@vitorino.net.br

**Comment by Emiliano** [comment]
Erick - eu@erick.tech

**Comment by tresDaniel** [comment]
Daniel - tres.daniel.s@gmail.com

**Reply by tresDaniel** [reply]
já tenho! s2

**Comment by thukabjj** [comment]
Arthur - arthur.alvesdeveloper@gmail.com

**Comment by jnassula** [comment]
Jonata - jonatanassula@gmail.com

**Reply by jnassula** [reply]
já tenho

**Comment by rodrigolosik** [comment]
Rodrigo Losik - rodrigolosik@gmail.com
